main_test.py

```
@@ -1,4 +1,4 @@
-from main import *
+from main import Human
```

Ruff (0.8.2)

`main.Human` imported but unused

Remove unused import: `main.Human`

(F401)